ingestion/src/metadata/ingestion/ometa/ometa_api.py

```diff
@@ -571,3 +573,55 @@ def close(self):
             None
         """
         self.client.close()
+
+    def _get_formmated_table_name(self, table_name):
+        return table_name.replace("[", "").replace("]", "")
+
+    def _create_lineage(self, from_table, to_table, service_name):
```

Comment:
Also, please add type hints and meaningful names to the methods to help understand the scenario we are handling. This `_create_lineage` does only work for Table entities and passing some names that even need to be formatted.

This does not feel generic enough to be part of the lineage_mixin. Let's move this method to a utilities module and name it correctly and with proper docstring to understand what we are handling here